video/out/vo_gpu_next.c

```diff
@@ -644,7 +644,7 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
     if (!pl_swapchain_start_frame(p->sw, &swframe)) {
         // Advance the queue state to the current PTS to discard unused frames
         pl_queue_update(p->queue, NULL, &(struct pl_queue_params) {
-            .pts = frame->current->pts + frame->vsync_offset,
+            .pts = frame->current->pts,
```

Comment:
Well this one doesn't actually matter of course. I only deleted it for consistency's sake.